funnel/forms/proposal.py

```diff
@@ -48,6 +48,8 @@ class ProposalLabelForm(forms.Form):
                     ],
                 ),
             )
+    if not project.labels:
+        return None
```

Comment:
Just `return` will do.

Reply:
Move this block right below the `def`. No need to define a class and iterate through labels if there are no labels.